src/ruleset.ts

```diff
@@ -29,6 +29,36 @@ export default {
             }
 
         },
+        "aep-131-response-schema": {
+            description: "Verifies that the response is an AEP resource.",
+            message: "The response body does not include an AEP resource.",
+            severity: DiagnosticSeverity.Error,
+            formats: [oas3],
+            given: "$.paths[*].get.responseBody.content[application/json].schema",
+            then:
+            {
+                function: schema,
+                functionOptions: {
+                    schema: {
+                        type: "object",
+                        required: ["x-aep-resource"],
+                    }
+                }
+            },
+        },
+        "aep-131-operation-id": {
+            description: "Verifies that the operation ID is of the form get{resource}",
+            message: "The operation ID does not conform to AEP-131",
```

Comment:
I wanted to revisit operationIDs and make sure we had the right syntax for those: https://github.com/aep-dev/aep.dev/issues/208. Could we wait on this?